util/template/expression_template_test.go

```diff
@@ -34,3 +34,50 @@ func Test_hasWorkflowParameters(t *testing.T) {
 		assert.False(t, hasWorkflowFailures("workflow .failures"))
 	})
 }
+
+func Test_hasVarInEnv(t *testing.T) {
```

Comment:
Please use the existing function naming convention of `TestHasVarInEnv`

Reply:
will change, I just kept it consistent with this file.